RELEASE_NOTES.md

```diff
@@ -1,7 +1,13 @@
+5.1.0
+=====
+- Add per-rollup metrics to DataCubeIo
+- Due to possible high cardinality, DataCubeIo constructors now require a boolean
+  controlling whether those metrics are maintained
+
 5.0.1
```

Comment:
well this is weird, there's no 5.0.1 release

Reply:
Oh, yep. The changes slated to be 5.0.1 are in https://github.com/urbanairship/datacube/pull/89 and not yet merged.